pkg/scheduler/scheduler.go

```diff
@@ -88,7 +88,10 @@ func (pc *Scheduler) Run(stopCh <-chan struct{}) {
 	pc.cache.SetMetricsConf(pc.metricsConf)
 	pc.cache.Run(stopCh)
 	pc.cache.WaitForCacheSync(stopCh)
-	klog.V(2).Infof("Scheduler completes Initialization and start to run")
+	if err := pc.cache.WaitForHandlersSync(stopCh); err != nil {
+		panic(fmt.Sprintf("failed to wait for handlers sync: %v", err))
```

Comment:
If we don't panic and go on, what will happen? Is that acceptable？

Reply:
The only way that error is returned is if stopCh cancel with ErrWaitTimeout. So we can make the method WaitForHandlersSync returnless and remove this panic.